extensions/mute-microphone/src/toggle-mute.tsx

```diff
@@ -3,13 +3,12 @@ import { getCurrentAudioInputLevel, toggleSystemAudioInputLevel } from "./shared
 
 export default async function toggleMute() {
   const currentAudioInputLevel = getCurrentAudioInputLevel();
+  await closeMainWindow();
   await toggleSystemAudioInputLevel(currentAudioInputLevel);
 
   try {
     await launchCommand({ name: "mute-menu-bar", type: LaunchType.Background });
   } catch {
     console.log("mute-menu-bar command is not active");
   }
```

Comment:
```suggestion
  }
  
  await closeMainWindow();
```

Reply:
I got why that sentence should go in the end, but in my test the toast doesn't work when `closeMainWindow()` it's at the end.

Reply:
Okay, seems like it affects the toast.